misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlParserErrorHandler.java

```diff
@@ -285,6 +312,7 @@ private void startContextIfRequired(ParserRuleContext currentCtx) {
             case TOML_TABLE:
             case KEY_VALUE_PAIR:
             case ARRAY_VALUE_LIST:
+            case INLINE_TABLE_LIST:
```

Comment:
In the toml parser, we start a context for the `INLINE_TABLE_START` but here in the error handler 
 we start a context for the `INLINE_TABLE_LIST`. Shouldn't both be the same?